fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/metadata/expressions/FunctionKeyExpression.java

```diff
@@ -299,7 +299,7 @@ public String getName() {
         }
 
         @Nonnull
-        abstract FunctionKeyExpression build(@Nonnull KeyExpression arguments);
+        public abstract FunctionKeyExpression build(@Nonnull KeyExpression arguments);
```

Comment:
It was previously impossible to build a `FunctionKeyExpression.Builder` outside of this package.